packages/edit-site/src/components/layout/index.js

```diff
@@ -296,82 +280,82 @@ export default function Layout() {
 
 					<SavePanel />
 
-					{ showCanvas && (
-						<>
-							{ isListPage && <PageMain /> }
-							{ isEditorPage && (
-								<div className="edit-site-layout__canvas-container">
-									{ canvasResizer }
-									{ !! canvasSize.width && (
-										<motion.div
-											whileHover={
-												isEditorPage &&
-												canvasMode === 'view'
-													? {
-															scale: 1.005,
-															transition: {
-																duration:
-																	disableMotion
-																		? 0
-																		: 0.5,
-																ease: 'easeOut',
-															},
-													  }
-													: {}
+					{ areas.content && canvasMode !== 'edit' && (
+						<div
+							className="edit-site-layout__area"
+							style={ {
+								maxWidth: widths?.content,
+							} }
+						>
+							{ areas.content }
+						</div>
+					) }
+
+					{ areas.preview && (
+						<div className="edit-site-layout__canvas-container">
```

Comment:
I'd appreciate it if we could consolidate the classes around the concept of "areas" as well. These are being used in a few places, so it can be done in a follow-up as well.

Reply:
I agree, the problem is that the "preview" area is a bit complex with animations and stuff so I was afraid of touching it. Could be a follow-up too.